.idea/thriftCompiler.xml

```diff
@@ -0,0 +1,6 @@
+<?xml version="1.0" encoding="UTF-8"?>
```

Comment:
Could you pls unstage this file? Or please clarify why this is required?

Reply:
I installed the thrift plugin, so the file is created automatically, I have  disabled the plugin and removed the file.